action.yml

```diff
@@ -20,6 +23,15 @@ outputs:
 runs:
   using: 'composite'
   steps:
+    uses: astral-sh/setup-uv@v6
+    with:
+      version: ${{ inputs.uv-version }}
+      # we just cache the venv-dir directly
+      enable-cache: false
+
+    # we prefer to use github-managed pythons rather than uv pythons:
+    # it's closer to production and there's been issues trying to get uv pythons
+    # to work with pyo3 (https://github.com/getsentry/streams/pull/158)
```

Comment:
I don't think you need this here to make your POC -- and you should freeze the version (you can just put this in sentry for now)